examples/compose/plot_column_transformer_mixed_types.py

```diff
@@ -87,6 +87,13 @@
 clf.fit(X_train, y_train)
 print("model score: %.3f" % clf.score(X_test, y_test))
 
+###############################################################################
+# HTML representation of ``Pipeline``
+###############################################################################
+# When the ``Pipeline`` is printed out in a jupyter notebook an HTML
+# representation of the estimator is displayed as follows:
```

Comment:
This is great !
Is there a way to get this display outside a jupyter notebook ?
I would also add this whole paragraph on the other examples in `examples/compose`.

Reply:
> Is there a way to get this display outside a jupyter notebook ?

Currently, the private method `_estimator_repr_html` will output the HTML which can be used to allow users to embedding things in any webpage. We can make this public in future.

In what other contexts do you see this display being used?

Reply:
Some people never use notebooks (e.g. myself), but could find useful to visualize their pipeline, e.g. exporting the html to a file, or directly opening it in a web browser tab.

Why would you not make a public function?

Reply:
> Why would you not make a public function?

I have been fairly conservative with making functions public. If there is a use case, like for exporting to a HTML file in your case, then it makes more sense to make it public.

I would prefer not to handle file handling in `estimator_repr_html`, does the standard way to do IO in python work for you?

```py
with open('est.html', 'w') as f:
    f.write(estimator_repr_html(est))
```

Reply:
This would probably be enough yes !

Reply:
While updating the docs for making this public, I starting to consider asking the following API (which currently works):

```py
with open('est.html', 'w') as f:
    f.write(est._repr_html_())
```

This would be a bit less discoverable, but it avoids defining a public function. What do you think?

Reply:
Given that `_repr_html_` seems already a standard API in IPython, I think this is reasonable.